### PR TITLE
Add jest framework

### PR DESCRIPTION
**Pull request by maheshschand**
This resolves issue #42 .

### CODE REVIEW

**Comment by maheshschand** [comment]
@rajatkb Please review this.

File: Notifier-service/package.json [hunk 1]
```diff
@@ -4,23 +4,28 @@
...
     "start": "ts-node-dev --respawn src/server.ts",
     "run": "node build/server.js"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "body-parser": "^1.19.0",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "jest": "^25.1.0",
```
Comment (rajatkb) [inline Notifier-service/package.json]: * It's a dev dependency. move it there.

File: Notifier-service/package.json [hunk 2]
```diff
@@ -4,23 +4,28 @@
...
     "mongoose": "^5.9.3",
+    "sqlite3": "^4.1.1",
```
Comment (rajatkb) [inline Notifier-service/package.json]: * Why needed ?

File: Notifier-service/package.json [hunk 3]
```diff
@@ -4,23 +4,28 @@
...
+    "supertest": "^4.0.2",
```
Comment (rajatkb) [inline Notifier-service/package.json]: * Why needed ?

File: Notifier-service/package.json [hunk 4]
```diff
@@ -4,23 +4,28 @@
...
     "winston": "^3.2.1"
   },
   "devDependencies": {
     "@types/express": "^4.17.2",
     "@types/mongoose": "^5.7.3",
     "@types/node": "^13.7.6",
+    "cross-env": "^7.0.2",
```
Comment (rajatkb) [inline Notifier-service/package.json]: * This is for ?

File: Notifier-service/package.json (hunk 3 above)
Comment (maheshschand) [reply Notifier-service/package.json]: While testing for unit tests, I tried using this frameworks. Sorry, I will remove it.

**Comment by maheshschand** [comment]
@rajatkb I made necessary changes.

File: Notifier-service/jest-test/sample.test.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+describe('Sample Test', () => {
```
Comment (rajatkb) [inline Notifier-service/jest-test/sample.test.js]: Make it compatible to Typescript.  Look into Jest-typescript

**Comment by rajatkb** [comment]
No response from @maheshschand  , and this being a issue that dev were dependent on is resolved. @maheshschand  you can work on including useful test if you wish.

**Comment by maheshschand** [comment]
Extremely sorry for the late response @rajatkb . I am having some classwork to do. So, I couldn't respond to you. I will make necessary changes according to the repo and your requests.

**Comment by rajatkb** [comment]
Ok it's fine. I have included the jest framework. So it will be great if you can identify some places in the project where test will be necessary.

**Comment by maheshschand** [comment]
OK. For sure.

**Pull request closed without merging**